Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppAcquireTokenViewController.m

```diff
@@ -590,7 +589,7 @@ - (void)acquireTokenInteractive:(id)sender
 
     if (_enableClientCapabilities.selectedSegmentIndex == 1)
     {
-        capabilities = @[AD_CLIENT_CAPABILITY_LLT];
+        capabilities = @[@"cp1"];
```

Comment:
Just curious, llt is not no longer allowed for the manual testing? :)

Reply:
"llt" got changed to "cp1" :)